tests/src/ut_test_kvp.c

```diff
@@ -471,6 +471,159 @@ void test_ut_kvp_string(void)
 
 }
 
+void test_ut_kvp_dataByte( void )
+{
+    int bytes_count = 0;
+    unsigned char *output_bytes;
+    test_ut_memory_t kvpdata;
+
+    /* Check for INVALID_PARAM */
+    UT_LOG_STEP("ut_kvp_getDataBytes() - Check for INVALID_PARAM");
+    output_bytes = ut_kvp_getDataBytes(NULL, "decodeTest/checkBytesSpace", &bytes_count);
+    UT_ASSERT(output_bytes == NULL );
+    UT_ASSERT(bytes_count == 0);
+
+    output_bytes = ut_kvp_getDataBytes(gpMainTestInstance, NULL, &bytes_count);
```

Comment:
you should add comments to state what test your doing for each, so it's clear.

Reply:
I have added UT_LOG_STEP now, to make it clear.